pages/caps/taxadeabandono/index.js

```diff
@@ -258,18 +267,29 @@ const TaxaAbandono = () => {
                   ...getPropsFiltroPeriodo(
                     periodos,
                     filtroPeriodoCID,
-                    setFiltroPeriodoCID
+                    setFiltroPeriodoCID,
+                    false
                   )
                 } />
               </div>
             </div>
 
             { loadingCID
-              ? <Spinner theme='ColorSM' height='70vh' />
-              : <ReactEcharts
-                option={ getOpcoesGraficoCID(agregadosPorCID) }
-                style={ { width: "100%", height: "70vh" } }
-              />
+              ? <Spinner theme="ColorSM" height="70vh" />
+              : <div className={ styles.GraficoCIDContainer }>
+                <ReactEcharts
+                  option={ getOpcoesGraficoCID(agregadosPorCID) }
+                  style={ { width: "50%", height: "70vh" } }
+                />
+
+                <TabelaCid
+                  labels={ {
+                    colunaCid: "Grupo de diagnósticos",
+                    colunaQuantidade: "Abandonaram no mês",
```

Comment:
Pode aceitar essa sugestão de mudança, por favor? Sei que está como abandono no pdf que te passei, mas o nome da página mudou de Taxa de abandono para Taxa de não adesão, com isso algumas palavras usadas no novo painel mudaram

```suggestion
                    colunaQuantidade: "Evadiram no mês",
```